### ISSUE TITLE
Tab icons all desaturated

### ISSUE THREAD

**Issue by ryanprior** [Zonnev/elementaryos-firefox-theme#96]
Hi there! Lately all my tab icons are gray-scale unless they're active. Color in the icons helps me identify which tab I want. The elementary HIG has an entry on this: https://elementary.io/docs/human-interface-guidelines#colored-vs-symbolic

Can we review the decision of whether to show color tab icons?

**Comment by Zonnev** [comment]
Hello, I know this is a problem for some. My goal was to distinguish the active tab from the other tabs but maybe I overly done that. I have tried to dim the colours of the icons of the non active tabs a bit but still keep the colours:
![Schermafdruk van 2020-11-23 12 34 54](https://user-images.githubusercontent.com/32688765/99957738-78c65c80-2d88-11eb-9765-48db0040f68b.png)
What do you think about this solution?

**Comment by ryanprior** [comment]
Yeah I like that better. The color strip at the top of the tab was really helpful too though - do you think it will be possible to use the accent color when that becomes available in elementary 6, and put that color above the active tab? (People who choose a gray accent color would then get the current behavior.)

**Comment by Zonnev** [comment]
I didn't really liked the blue colour strip above the tabs because I like the tabs to be as close as possible to the elementary dark theme so that blue strip was bothering me. I made it a subtle gray to still have a sort of focus but I won't go back to the blue strip anymore.

The tabs on Odin have the same kind of gray border as I made for the Hera theme but on Odin this is the default tab style, on Hera I added it to create more focus. So, on Odin there is also no need to use a coloured strip.
